hydromt/models/components/grid.py

```diff
@@ -762,19 +762,6 @@ def bounds(self) -> List[float]:
         if len(self.grid) > 0:
             return self.grid.raster.bounds
 
-    @property
```

Comment:
This was necessary since it was causing an error "cannot override property" or something to that tune hence it was. I think it's best to find a way around that in a new pr if that's okay.

Reply:
Oh true! Well was not too complex to derive and we can always check main to see what is was when we fix it later.